setup.py

```diff
@@ -26,9 +26,12 @@
 .. _`Github`: https://github.com/GetStream/stream-python
 '''
 
+requests = 'requests[security]>=2.4.1' if sys.version_info < (2, 7, 9) else (
```

Comment:
It already was specified that way, you want me to limit all the deps to a specific major version?

Another option is to have travis run pip install with --upgrade that way the daily runs will notify us of any breakage from deps.